.github/workflows/main.yaml

```diff
@@ -0,0 +1,32 @@
+name: Automated testing
+
+on:
+  push:
+    branches:
+      - "*"
+      - "!test*"
+jobs:
+  run-checks:
+    runs-on: ubuntu-latest
+    outputs:
+      cache_key: ${{ steps.texlive.outputs.cache_key }}
```

Comment:
Output `cache_key` is never used (and there's no step named `texlive` in current workflow).
```suggestion
```